include/git2/rebase.h

```diff
@@ -30,19 +30,32 @@ typedef struct {
 	unsigned int version;
 
 	/**
-	 * Provide a quiet rebase experience; unused by libgit2 but provided for
-	 * interoperability with other clients.
+	 * Used by `git_rebase_init`, this will instruct other clients working
+	 * on this rebase that you want a quiet rebase experience, which they
+	 * may choose to provide in an application-specific manner.  This has no
+	 * effect upon libgit2 directly, but is provided for interoperability with
+	 * other clients.
 	 */
 	int quiet;
 
 	/**
-	 * Canonical name of the notes reference used to rewrite notes for
-	 * rebased commits when finishing the rebase; if NULL, the contents of
-	 * the coniguration option `notes.rewriteRef` is examined, unless the
-	 * configuration option `notes.rewrite.rebase` is set to false.  If
-	 * `notes.rewriteRef` is NULL, notes will not be rewritten.
+	 * Used by `git_rebase_finish`, this is the canonical name of the notes
+	 * reference used to rewrite notes for rebased commits when finishing the
```

Comment:
"canonical name" is a term which only really libgit2sharp uses and it is migrating away from it. Could we leave it as "name"?